packages/dxf-deserializer/test/test-2d-entities.js

```diff
@@ -0,0 +1,141 @@
+const fs = require('fs')
+const path = require('path')
+const test = require('ava')
+const { CSG, CAG } = require('@jscad/csg')
+
+import {nearlyEqual} from '../../../test/helpers/nearlyEqual'
```

Comment:
this is an es6 import , to be avoided, please use require()

Reply:
Ok. Will change...

Reply:
Changes made.